siliconcompiler/tools/openroad/scripts/sc_procs.tcl

```diff
@@ -228,9 +244,32 @@ proc sc_supply_nets {} {
 
 proc sc_psm_check_nets {} {
   global openroad_psm_enable
+  global openroad_psm_skip_nets
 
   if { $openroad_psm_enable == "true" } {
-    return [sc_supply_nets]
+    set skip_nets []
+    foreach net_pattern $openroad_psm_skip_nets {
+      foreach skip_net [sc_find_net_regex $net_pattern] {
```

Comment:
Why do you first gather for the `skip_nets` list and then filter the `sc_supply_nets` using it?
Is it not possible to filter directly on the `sc_supply_nets`?

Reply:
Add good point, I'll switch it around